interleague3 .yaml

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
I don't think you need this file since you also have `interleague3.yaml`

Reply:
Yep we are going to delete that!